pydrad/parse/parse.py

```diff
@@ -21,13 +21,29 @@
 __all__ = ['Strand', 'Profile', 'InitialProfile']
 
 
-def get_master_time(hydrad_root):
-    log.debug('Reading master time array from all AMR files')
+def get_master_time(hydrad_root, read_from_cfg=None):
     amr_files = glob.glob(os.path.join(hydrad_root, 'Results/profile*.amr'))
-    time = np.zeros((len(amr_files),))
-    for i, af in enumerate(amr_files):
-        with open(af, 'r') as f:
-            time[i] = f.readline()
+    if read_from_cfg is None:
+        log.debug('Reading master time array from all AMR files')
+        time = np.zeros((len(amr_files),))
+        for i, af in enumerate(amr_files):
+            with open(af, 'r') as f:
+                time[i] = f.readline()
+    else:
+        log.debug('Creating master time array from config files')
+        try:
+            with open('HYDRAD/config/hydrad.cfg', 'r') as f:
+                lines = f.readlines()
+            cadence = float(lines[3])
+            with open(amr_files[0]) as f:
+                start_time = float(f.readline())
+            time = np.fromiter([start_time + t * cadence for t in range(len(amr_files))], dtype='float')
+        except FileNotFoundError:
+            log.debug('HYDRAD/config/hydrad.cfg not found -- reading AMR instead')
+            time = np.zeros((len(amr_files),))
+            for i, af in enumerate(amr_files):
+                with open(af, 'r') as f:
+                    time[i] = f.readline()
```

Comment:
```suggestion
    if read_from_cfg:
        log.debug('Creating master time array from config files')
        with open(os.path.join(hydrad_root, 'HYDRAD/config/hydrad.cfg'), 'r') as f:
            lines = f.readlines()
        cadence = float(lines[3])
        with open(amr_files[0]) as f:
            start_time = float(f.readline())
        time = start_time + np.arange(len(amr_files)) * cadence
    else:
        log.debug('Reading master time array from all AMR files')
        time = np.zeros((len(amr_files),))
        for i, af in enumerate(amr_files):
            with open(af, 'r') as f:
                time[i] = f.readline()
```

I don't think we need to fall back to the AMR behavior if the cfg option doesn't work. I don't know of a case where the `hydrad.cfg` file would be missing and if it is, I think it would be good to make the user aware of this.

Reply:
I followed this for the most part, but I think there ought to be a try-except statement for reading it.  It's possible to be missing, so the error catching still seems useful.  We sometimes pass only the Results folder for sharing results.

Reply:
Or is there a better way to alert the user?

Reply:
Ah right. I hadn't though of that use case. That's fair though when you do the `with` statement, it will throw a `FileNotFoundError`(or something similar)  and then it is just a matter of setting the `read_from_cfg` keyword to False (this is the default anyway)

Reply:
I'm confused.  Do we want to keep the try-except?  I pushed a commit yesterday including it, but it's easy enough to remove.

Reply:
I'd prefer to not have it. If you're ok with that, you can just hit the "commit" button below my suggestion and it will automatically apply that change to this PR. I'll run the hydrad tests locally one more time before we merge just to double check everything still works as expected.

Reply:
Can't use that because this is outdated.  I just did it manually.